internal/services/notificationhub/notification_hub_namespace_resource_test.go

```diff
@@ -40,15 +40,15 @@ func TestAccNotificationHubNamespace_updateTag(t *testing.T) {
 				check.That(data.ResourceName).Key("tags.%").HasValue("1"),
 			),
 		},
-		data.ImportStep(),
+		data.ImportStep("namespace_type"),
```

Comment:
Why do we need to add `namespace_type` here? Is it not returned by the API or is it not set in the read function?

Reply:
It's not returned, and refs https://github.com/hashicorp/terraform-provider-azurerm/blob/main/internal/services/notificationhub/notification_hub_namespace_resource_test.go#L28

Reply:
other tests also do it